Userland/Games/Hearts/MainWidget.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2024, Sanil Gupta <sanilg566@gmail.com>.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+#pragma once
```

Comment:
Welcome to the project! The lint check step appears to be failing here; just add a newline between these 2 lines and you should be good to go 👍